README.md

```diff
@@ -79,9 +79,10 @@ To-do:
 
 # How do I configure what gets copied to my output directory?
```

Comment:
Probably should warn before this line (80) that the following is about Windows (and WindowsRT) only.

Reply:
Or create a new page for platform specific info? What do you think?

Reply:
As discussed in private : It's not just about windows, it will be about linux too, if it's done one day. I'm ok to split the files, I'm doing it now, but by topic rather than platform.